resource-server-plugin/build.gradle

```diff
@@ -0,0 +1,12 @@
+description = "Maven Plugin for the Resource Server project."
+
+dependencies {
+    compile project(':resource-server-core')
+    compile project(':resource-server-api')
+
+    compile "org.slf4j:slf4j-api:${slf4jVersion}"
+    compile "org.slf4j:jcl-over-slf4j:${slf4jVersion}"
+    compile "org.apache.maven:maven-plugin-api:3.5.2"
```

Comment:
:question: could the version number be pulled from `gradle.properties`?

Reply:
yes, will do